org.eclipse.debug.core/core/org/eclipse/debug/internal/core/LaunchManager.java

```diff
@@ -536,6 +537,21 @@ public static Document getDocument() throws ParserConfigurationException {
 	 * @throws IOException if I/O error occurs
 	 */
 	public static String serializeDocument(Document doc) throws TransformerException, IOException {
+		return serializeDocument(doc, System.lineSeparator());
+	}
+
+	/**
+	 * Serializes a XML document into a string - encoded in UTF8 format, with
+	 * specified line separator.
+	 *
+	 * @param doc document to serialize
+	 * @param lineDelimiter the new line separator to use
+	 * @return the document as a string
+	 * @throws TransformerException if an unrecoverable error occurs during the
+	 *             serialization
+	 * @throws IOException if I/O error occurs
+	 */
+	public static String serializeDocument(Document doc, String lineDelimiter) throws TransformerException, IOException {
```

Comment:
This has a javadoc warning as lineDelimeter is not defined as argument

Reply:
> /@karypid The API has slightly changed. Instead of `ResourcePlugin.getLineSeparator(file)` you now need to use `file.getLineSeparator()` (make sure you have latest source or binaries for org.eclipse.core.resources available in your workspace or target-platform to see this change). Can you please update your PR accordingly?

Ok, I will do this when pushing the unit tests. Will probably update tomorrow.

Reply:
This is done.